app/views/vendors/new.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<h1>Add New Vendor!</h1>
+
+<section class = "form-container">
+  <span><%=form_for @vendor do |f| %></span><br>
+  <span>Name:<%=f.text_field :name %></span><br>
+  <span>Num. Employee:<%=f.text_field :no_of_employees %></span><br>
+  <%= f.hidden_field :market_id, :value => (params[:market_id]) %>
+  <span><%=f.submit%></span>
+  <% end %>
```

Comment:
Try not to use `br` in your html - styling should be done with css. It may be easier in the short-term, but it makes future style changes difficult on yourself/other future devs working on the code.